docs/configuring-metabase/caching.md

```diff
@@ -11,9 +11,11 @@ If your question results don't change frequently, you may want to store the resu
 
 For example, if your data only updates once a day, there's no point in querying the database more than once a day, as the data won't have changed. Returning cached results can be significantly faster, as the database won't have to recompute the results to load your question.
 
+You can set [caching invalidation policies](#cache-invalidation-policies) for [questions](#question-caching-policy), [dashboards](#dashboard-caching-policy), and [databases](#database-caching-policy).
+
 ## Cache invalidation policies
 
-These policies determines how long cached results will be stored.
+These policies determine how long cached results are valid.
```

Comment:
In the recording I say that when the cache is invalidated, it is not deleted - no, I was wrong, the cache is deleted when it is invalidated. That's why I wondered if "clearing" or "deleting" the cache is friendlier than the comp-sci term "invalidation".